src-tauri/src/plugins/mod.rs

```diff
@@ -139,9 +140,9 @@ pub async fn initialise_plugin(path: &path::PathBuf) -> anyhow::Result<()> {
 		devices.push(device.into());
 	}
 
-	let code_path = code_path.unwrap();
+	let code_path = code_path.unwrap().to_lowercase();
```

Comment:
Ooh, I meant more like making it lowercase inside the .ends_with checks (`if code_path.to_lowercase().ends_with(...)`) - making it lowercase here could stop some plugins from working on case-sensitive filesystems.

Reply:
Oh I see, my bad !

Reply:
@ninjadev64 Let me know if the changes are what you meant.